src/video/windows/SDL_windowsevents.c

```diff
@@ -1757,6 +1776,7 @@ WIN_PumpEvents(_THIS)
                 /* This mouse message happened after the warp */
                 SDL_last_warp_time = 0;
             }
+#endif !defined(__XBOXONE__) && !defined(__XBOXSERIES__)
```

Comment:
GCC shows warning: extra tokens at end of #endif directive

```suggestion
#endif /*!defined(__XBOXONE__) && !defined(__XBOXSERIES__)*/
```

Reply:
Oh, yeah that should be a comment was just a copy/paste error

Reply:
If someone sends this in I'll merge it right away.

Reply:
Ok put the change in a PR just now